tests/keras/utils/data_utils_test.py

```diff
@@ -368,5 +370,33 @@ def test_finite_generator_enqueuer_processes():
     enqueuer.stop()
 
 
+def test_prepare_sparse_sliced_data():
+    test_train_data = mx.test_utils.rand_ndarray((10, 10), 'csr', 0.01)
```

Comment:
use numpy to create random data of (10,10) and remove import mxnet, this is a general until function, should be tested with numpy and should not rely on mxnet

Reply:
fixed